extensions/librdkafka/docker_tests/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+#
```

Comment:
> These are manual tests that are ran for the processor. I would not want a PR with a processor that does not involve the testing done for it.

But I can't see any docker tests in this PR.  We have a CMakeLists file but the list of source files it applies to is empty.

Reply:
I mean these will sit in docker, but right now they are just something to run manually on minifi with a working local broker. I will soon get to start working on dockerized tests, that PR will probably get rid of these.

Reply:
I still think that the future PR in which you dockerize the tests would be a better place for this CMakeLists file, and having a CMakeLists file in a directory by itself is strange, but since this is temporary, I'm OK with it.